charts/simple-app/templates/_helpers.tpl

```diff
@@ -11,3 +11,11 @@ setting.
 {{- .Values.proxySidecar.image.repository }}:{{ $tag }}
 {{- end }}
 {{- end }}
+
+{{- define "nd-common.extendedLabels" -}}
+{{- $baseLabels := include "nd-common.labels" . | fromYaml -}}
+{{- $extendedLabels := merge $baseLabels (dict
+    "helm.chart.simple-app/version" .Chart.Version
```

Comment:
I think we should just do something like. `helm.chart/version: ...` and `helm.chart/name: ...`